utils/scylla_doctor.py

```diff
@@ -45,6 +45,8 @@ def version(self):
         return version
 
     def install_scylla_doctor(self):
+        if self.node.parent_cluster.cluster_backend == "docker":
+            self.run('apt update')
```

Comment:
it should be with sudo, I think..

Reply:
Yes, it will run as sudo:

https://github.com/scylladb/scylla-cluster-tests/blob/d3021bbd2a4ecadab827a75060b2eeab679c2c45/utils/scylla_doctor.py#L29

Reply:
o.k. it's a bit confusing, cause we use to see run vs. sudo commands.